CHANGELOG.md

```diff
@@ -1,3 +1,143 @@
+# 84.0.0
+
+In this release we corrected a text field accesssibility behavior added ripple to chips and added
+additional delegate calls to post bottom sheet's scrolling.
+
+We finished updating additional theming docs for multiple components including chips, dialogs, cards.
+
+## Breaking changes
+
+### Text Field
+
+We fixed the accessibilty behavior of MDCTextField. We were using the value instead of the label.
```

Comment:
> We fixed the accessibilty behavior of MDCTextField

This seems a bit broad; consider deleting.

Reply:
Maybe something like this?
>We modified the VoiceOver behavior of MDCTextField such that much of the information previously returned by `-accessibilityValue` is now being returned by `-accessibilityLabel`.